stylesheets/print.scss

```diff
@@ -0,0 +1,52 @@
+@page {
+  margin-top: 1.5cm;
+  margin-right: 1cm;
+  margin-left: 1cm;
+  margin-bottom: 2cm;
+}
+
+body {
+  font: 11pt Georgia, "Times New Roman", Times, serif;
```

Comment:
By using the `font` shorthand here, I think you're inadvertently overriding some styles from the base (all.css) stylesheet, like `line-height`. As a result, bullet points are a bit too close together (and even overlapping when they have inline code). Can you replace this with separate `font-size` and `font-family` declarations.